src/store/modules/shipment/mutations.ts

```diff
@@ -8,6 +8,13 @@ const mutations: MutationTree <ShipmentState> = {
   },
   [types.SHIPMENT_CURRENT_UPDATED] (state, payload) {
     state.current = payload.current;
-  }
+  },
+  [types.SHIPMENT_UPDATE_PRODUCT_COUNT] (state, payload) {
+    state.current.items.find((item: any) => {
```

Comment:
I think we should do it in action instead and use`SHIPMENT_CURRENT_UPDATED` or mutation to just update current items

Reply:
Yes Sir, updated the code to use SHIPMENT_CURRENT_UPDATED